src/tools/gophish_complete.py

```diff
@@ -3,24 +3,24 @@
 """Complete a campaign in GoPhish and/or output a GoPhish campaign summary.
 
 Usage:
-  gophish-complete (--auto=CAMPAIGN_ID | --complete | --summary ) [--log-level=LEVEL] SERVER API_KEY
+  gophish-complete (--complete | --summary) [--Campaign=NAME] [--log-level=LEVEL] SERVER API_KEY
   gophish-complete (-h | --help)
   gophish-complete --version
 
 Options:
   API_KEY                   GoPhish API key.
   SERVER                    Full URL to GoPhish server.
-  -a --auto=CAMPAIGN_ID     Complete a specific campaign identified by CAMPAIGN_ID.
-  -c --complete             Display a list of campaigns to choose which to complete.
-  -s --summary              Choose a campaign from a list to output a summary.
+  -C --Campaign=NAME        GoPhish campaign name.
```

Comment:
We don't usually do capital letter arguments.  As I think about this more, our args can be further simplified:

```
gophish-complete [--campaign=NAME] [--summary-only] [--log-level=LEVEL] SERVER API_KEY
```
The default behavior will be to complete a campaign, so no need to have the `--complete` arg at all.  The optional behavior will be triggered via `--summary-only`.  What do you think?

Reply:
As discussed, I think this is a good idea. I am working on making the needed changes.

Reply:
Changes made to adjust the functionality in commit 9658acd. 

I also adjusted `complete_campaign` so it will raise a warning if GoPhish returns the completion as unsuccessful.  This change allowed me to remove many of the conditional statements and repetitive setting of `success`.